diff --git a/cpuset/commands/set.py b/cpuset/commands/set.py
index fee03b1..3aa330e 100644
--- a/cpuset/commands/set.py
+++ b/cpuset/commands/set.py
@@ -412,8 +412,8 @@ def modify(name, cpuspec=None, memspec=None, cx=None, mx=None):
     log.debug('modifying cpuset "%s"', nset.name)
     if cpuspec: nset.cpus = cpuspec
     if memspec: nset.mems = memspec
-    if cx: nset.cpu_exclusive = cx
-    if mx: nset.mem_exclusive = mx
+    # if cx: nset.cpu_exclusive = cx
+    # if mx: nset.mem_exclusive = mx
 
 def active(name):
     """check that cpuset by name or cset is ready to be used"""
